modules/net-lb-app-ext/main.tf

```diff
@@ -28,6 +28,12 @@ locals {
     [for k, v in google_compute_ssl_certificate.default : v.id],
     [for k, v in google_compute_managed_ssl_certificate.default : v.id]
   )
+  trust_config_name = "trust-config-${var.name}"
+  mtls_policy_name  = "mtls-policy-${var.name}"
+}
+
+data "google_project" "project" {
```

Comment:
we avoid data sources as it makes it hard to test modules, can we derive the number from some other id already in the module, or ask the user to pass it in?